book/configuration.md

```diff
@@ -22,8 +24,32 @@ model.fit(variables_train, outcome_train)
 prediction = model.predict(variables_test, constant_a=7, max_v=1000)
 
 prediction.to_csv("outputs/predictions.csv")
+
+```
+
+```{code-tab} r R
+data <- read.csv("C:/a/very/specific/path/to/input_data.csv") 
```

Comment:
add a note to top of each saying that it's not good practice

Reply:
Fixed.